### DISCUSSION TITLE
Responsivness on changing size screen

### DISCUSSION THREAD

**Discussion by dbwebapp** [BitCannaGlobal/WeedWallet#99] [General]
when you change the size of the window and put it back, the graph still take 100% of the horizontal space . It should come back to 50%. 

https://github.com/BitCannaGlobal/WeedWallet/assets/96944913/3b4a0272-39f1-4993-8083-997b3da1af03

**Comment by TJ-BitCanna** [comment]
Yeap, we will look into responsiveness also in the new design.

**Comment by TJ-BitCanna** [comment]
Solved for the graph. All other responsiveness issues we will take into account when testing page-by-page
